Sources/PackagePlugin/Plugin.swift

```diff
@@ -213,60 +246,9 @@ extension Plugin {
     }
 }
 
-/// Message channel for communicating with the plugin host.
+/// Message channel for bidirectional communication with the plugin host.
 internal fileprivate(set) var pluginHostConnection: PluginHostConnection!
 
-/// A message that the host can send to the plugin.
-enum HostToPluginMessage: Decodable {
```

Comment:
These are moved out into `PackageMessages`, which also contains the Codable structures that are used for all communication between the plugin and the host.